docs/walk-through/README.md

```diff
@@ -0,0 +1,114 @@
+# "Step-by-step Walk-Through" or 
+# How to Use a Hardware Wallet with Bitcoin Core Wallet
+
+Summary: On this page we describe step-by-step and show in screenshots how to use a hardware wallet and HWI 
+together with a Bitcoin Core Wallet. As hardware wallet example we have used a Trezor.
+
+## Create a watch-only Bitcoin Core wallet for Trezor
+Create your watch-only Bitcoin Core Wallet as well described in [Using Bitcoin Core with Hardware Wallets](../bitcoin-core-usage.md). 
+You find all the details in this link. But in summary, one opens one terminal and runs `bitdoind`. E.g.
+
+```
+bitcoind -testnet  -datadir=$HOME/.bitcoin-testnet -rpcallowip=127.0.0.1 -rpcuser=test -rpcpassword=somedifficultpassword
+```
+for a testnet `bitcoind` daemon, or
+```
+bitcoind -rpcallowip=127.0.0.1 -rpcuser=test -rpcpassword=somedifficultpassword
+```
+for a mainnet, i.e. regular, `bitcoind` daemon.
+
+Then in another terminal run commands similar to these, adapted to your environment:
+```
+hwi.py enumerate # this shows you the fingerprint of hardware wallet
+TREZORFINGERPRINT_TESTNET="yourTrezorFingerprint" # shown by "hwi enumerate"
+# in this example we use SEGWIT BECH32 ADDRESSES
+DERIVATIONPATH_TESTNET=1 # testnet uses derivation paths like m/84h/1h/0h/0/* and m/84h/1h/0h/1/* on testnet
+DERIVATIONPATH_MAINNET=0 # mainnet uses derivation paths like m/84h/0h/0h/0/* and m/84h/0h/0h/1/* on testnet
+# if the mainnet path is used on testnet, it will work too but Trezor device gives warnings 
+# of unknown address on Trezor display
+wallet=wallet.test
+rec=$(hwi --testnet -f $TREZORFINGERPRINT_TESTNET getkeypool --wpkh --path m/84h/${DERIVATIONPATH_TESTNET}h/0h/0/* --keypool 0 1000)
+chg=$(hwi --testnet -f $TREZORFINGERPRINT_TESTNET getkeypool --wpkh --path m/84h/${DERIVATIONPATH_TESTNET}h/0h/1/* --keypool --internal 0 1000)
+while ! bitcoin-cli -testnet -rpcuser=test -rpcpassword=somedifficultpassword -rpcclienttimeout=0 createwallet "$wallet" true ; do sleep 60; done
```

Comment:
It's unnecessary to specify `-rpcuser`, `-rpassword`, and `-rpcclienttimeout`. If you are specifying the datadir (as you did in your `bitcoind` command), then specifying `-datadir` for `bitcoin-cli` is sufficient.

I would suggest that for this guide, we avoid specifying addiional command line options for `bitcoind` and `bitcoin-cli`.

Reply:
OK, I removed the -rpcuser, -rpcpassword options. Ok, I will take out the -rpcclienttimeout option as well in all places.
Done in commit 8e6de745d625d5ebc481f0ccddf15e3dba91da3b.